docs/releases/2.0.txt

```diff
@@ -72,6 +72,9 @@ Minor features
 :mod:`django.contrib.postgres`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* A `distinct` keyword has been added to `ArrayAgg` to specify if array values 
```

Comment:
Follow the style of the 1.11 release notes: "The new distinct argument for ArrayAgg determines if aggregated values will be distinct." The backticks need to be doubled to have an effect on the rendered text.